oozie/src/main/java/org/apache/falcon/workflow/engine/OozieWorkflowEngine.java

```diff
@@ -959,13 +958,6 @@ private void performAction(String cluster, JobAction action, CoordinatorAction c
             status = Status.KILLED.name();
             break;
 
-        case IGNORE:
```

Comment:
This change is reversing the patch to ensure manually killed instances aren't retried.